seed/migrations/permitData.py

```diff
@@ -0,0 +1,102 @@
+
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import django.contrib.postgres.fields.jsonb
+import django_extensions.db.fields
+import autoslug.fields
+import django.utils.timezone
+import django.db.models.deletion
+from django.conf import settings
+
+class Migration(migrations.Migration):
```

Comment:
Inconsistent indentation in Python would not compile. 

General guidance on indentation:
https://www.python.org/dev/peps/pep-0008/#indentation